src/app.py

```diff
@@ -100,17 +103,43 @@ async def send_scene():
 
     await asyncio.gather(*future_sends)
 
+
```

Comment:
I think we should commit reformatting changes separately, this shouldn't be here